code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -25,7 +25,7 @@
 		qdel(O)
 	..()
 
-/obj/item/slime_extract/New()
+/obj/item/slime_extract/Initialize()
 		..()
```

Comment:
A: I'm doing this in #26386
B: if you convert something to initialize, you need to change the `..()` to `. = ..()`